.github/workflows/netlify_document_staging.yml

```diff
@@ -37,12 +37,7 @@ jobs:
         working-directory: docs/document/website
       - name: Install Netlify
         run: npm i netlify-cli@9 --no-save
+      - name: Install Netlify
```

Comment:
We only need to install netlify-cli once, so please remove the old installation command.
```suggestion
```